docs/csharp/language-reference/compiler-options/langversion-compiler-option.md

```diff
@@ -47,15 +47,21 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
   
 |Option|Meaning|  
 |------------|-------------|  
-|default|The compiler accepts all valid language syntax.|  
-|ISO-1|The compiler accepts only syntax that is included in the ISO/IEC 23270:2003 C# language specification.|  
-|ISO-2|The compiler accepts only syntax that is included in the ISO/IEC 23270:2006 C# language specification. This specification is available on the [ISO](http://go.microsoft.com/fwlink/?LinkId=144406) Web site.|  
-|3|The compiler accepts only syntax that is included in the version 3.0 [C# Language Specification](../../../csharp/language-reference/language-specification.md).|  
+|default|The compiler accepts all valid language syntax that it can support.|  
+|ISO-1|The compiler accepts only syntax that is included in the ISO/IEC 23270:2003 C# language specification. (net 2.x or newer)|  
+|ISO-2|The compiler accepts only syntax that is included in the ISO/IEC 23270:2006 C# language specification. This specification is available on the [ISO](http://go.microsoft.com/fwlink/? LinkId=144406) Web site. (net 3.x or newer)|
+|3|The compiler accepts only syntax that is included in the version 3.0 [C# Language Specification](../../../csharp/language-reference/language-specification.md). (net 4.x or newer)|
+|4|The compiler accepts only syntax that is included in the version 4.0 (net 4.x or newer)|
+|5|The compiler accepts only syntax that is included in the version 5.0 (net 4.x or newer)|
   
 ## Remarks  
  Metadata referenced by your C# application is not subject to **/langversion** compiler option.  
   
  Because each version of the C# compiler contains extensions to the language specification, **/langversion** does not give you the equivalent functionality of an earlier version of the compiler.  
+ 
+ While newer versions of the C# syntax do not explicitly depend on a specific version of the net framework runtime, compilers on older versions of the framework are not updated to support newer langauge syntax without additional configuration (which is beyond the scope of this article.) The above table lists the minimum net framework compiler version needed to compile for a given language version without any additional configuration needed.
+ 
+ Additionally, C# 6 and newer are not supported at all with this compiler, they require migrating to the newer .NET Compiler Platform ("Roslyn")
```

Comment:
I think this documentation should be about the latest version of `csc` (right now, Roslyn 2.2.0 supporting C# 7.0). I don't see why should it be only about the version of `csc` included in the framework or why it should talk about the old compilers included in the framework at all.